scripts/performance-data-migration/encounterMigration.sql

```diff
@@ -19,21 +20,16 @@ BEGIN
     SELECT encounter_type_id, uuid
     FROM input.encounter_type;
 
-  UPDATE input.encounter_type
-    SET creator = admin_id
-    WHERE creator IS NOT NULL;
-
-  UPDATE input.encounter_type
-    SET retired_by = admin_id
-    WHERE retired_by IS NOT NULL;
-
-  call debugMsg(1, 'input updated');
-
-  INSERT INTO encounter_type (name, description, creator, date_created, retired, retired_by,
+  INSERT INTO encounter_type (`name`, description, creator, date_created, retired, retired_by,
```

Comment:
Why have you added ` characters? Is it because "name" is syntax element in SQL?

Reply:
Yes